src/ui/public/accessibility/kbn_ui_ace_keyboard_mode.less

```diff
@@ -0,0 +1,24 @@
+@import (reference) "~ui/styles/variables";
+
+.ui-ace-keyboard-hint {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background: rgba(255, 255, 255, 0.7);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0;
+}
+
+.ui-ace-keyboard-hint-inactive {
+  display: none;
+}
+
+.ui-ace-keyboard-hint:focus {
+  opacity: 1;
+  border: 2px solid @globalColorBlue;
+  z-index: 1000;
+}
```

Comment:
Oh, sorry I meant efficient in terms of readability. By using  `&:focus` it's a bit easier to scan the styles and grok the context of all the available selectors -- they're all related to the "uiAceKeyboardHint" component.